ui/components/DatabaseSummary.js

```diff
@@ -154,6 +154,7 @@ const DatabaseSummary = (props) => {
           <Button
             type="submit"
             variant="contained"
+            data-test="database-reset"
```

Comment:
Hey @THEBOSS0369, curious if we could use data-testid vs data-test. Also can we clarify the name `database-reset-button`. We need to add more ids in future and this will clarify that this is for the button. 

- https://playwright.dev/docs/other-locators#id-data-testid-data-test-id-data-test-selectors
- https://playwright.dev/docs/locators#locate-by-test-id